instance_config.yaml

```diff
@@ -40,6 +40,9 @@ veda_deliverable_bucket:
 # Settings
 multi_upload_barrier: 2000000000
 
+# Ingest Secret
+veda_secret_access_key:
+veda_access_key_id:
```

Comment:
@muzaffaryousaf It's a patch until I can figure out what's wrong with the IAM roles. It's been a long term issue (and a ticket is in with AWS). I'll add a comment string.